### ISSUE TITLE
Add support for Terraform 1.1 syntax

### ISSUE THREAD

**Issue by ouranos** [terraform-linters/tflint#1275]
I'm trying to replace my `remote` block with the [new `cloud` block](https://www.terraform.io/docs/cli/cloud/settings.html#the-cloud-block) introduced in 1.1 but TFLint is not happy about it.

```hcl
terraform {
  cloud {
    organization = "my-org"

    workspaces {
      name = "my-workspace-name"
    }
  }
}
```

```
TFLint in staging-usw1/:
Failed to load configurations. 1 error(s) occurred:

Error: Unsupported block type

  on main.tf line 14, in terraform:
  14:   cloud {

Blocks of type "cloud" are not expected here.
```

## Version

```console
$ tflint -v
TFLint version 0.33.2
$ terraform -v
Terraform v1.1.0
on darwin_amd64
+ provider registry.terraform.io/hashicorp/aws v3.68.0
```

**Comment by bendrucker** [comment]
Ooh this is very interesting, the move from prefixes to tags. TFLint bundles Terraform so any new syntax requires package updates for compatibility.

**Comment by bendrucker** [comment]
`moved` is also new and super cool
